test/utils_test.go

```diff
@@ -400,8 +435,35 @@ func PullFromAssembler(t *testing.T, userConfig *armageddon.UserConfig, partyID
 			}
 		}
 
+		data := block.GetData().GetData()
+		blocksNumber := len(data)
+
 		totalBlocks++
-		totalTxs += uint64(len(block.GetData().GetData()))
+
+		if blocksNumber == 1 {
+			return nil
```

Comment:
can't there be a block with only one tx that is not the genesis? 
just ignore the genesis block